server/database/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM mysql:5.7.16
+FROM mysql:latest
```

Comment:
I wouldn't take the latest version.
I think it should be a selected version.

Reply:
Updated to 8.0.32